buildkite/src/Jobs/Release/MinaToolchainArtifactBullseyeArm64.dhall

```diff
@@ -0,0 +1,21 @@
+let MinaArtifactToolchain = ../../Command/MinaArtifactToolchain.dhall
```

Comment:
We are using bullseye toolchain for Ci everywhere. So we need to have arm64 copy too